toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -137,12 +142,19 @@ func (r *RpmRepoCloner) Initialize(destinationDir, tmpDir, workerTar, existingRp
 	//
 	// 2) Mount the directory to download RPMs into as a bind, allowing the chroot to write
 	// files into it.
-	overlayMount, overlayExtraDirs := safechroot.NewOverlayMountPoint(r.chroot.RootDir(), overlaySource, chrootLocalRpmsDir, existingRpmsDir, overlayUpperDirectory, overlayWorkDirectory)
+	outRpmsOverlayMount, overlayExtraDirs := safechroot.NewOverlayMountPoint(r.chroot.RootDir(), overlaySource, chrootLocalRpmsDir, existingRpmsDir, overlayUpperDirectoryRpms, overlayWorkDirectoryRpms)
 	extraMountPoints := []*safechroot.MountPoint{
-		overlayMount,
+		outRpmsOverlayMount,
 		safechroot.NewMountPoint(destinationDir, chrootDownloadDir, bindFsType, safechroot.BindMountPointFlags, bindData),
 	}
 
+	// Optionally include the special toolchain package directory if it is provided
+	if len(chrootLocalToolchainDir) > 0 {
```

Comment:
This looks fishy. The check doesn't make sense, since `chrootLocalToolchainDir` is a constant. Also, I'd expect the toolchain to always be pre-populated before this tool is run.

Reply:
Good catch. This was meant to be `toolchainRpmsDir`. I was thinking its reasonable that the cloner may be used in the future without a full toolchain available. I'll go ahead and drop it though, because at the rework to split the tools up more we were considering back then hasn't gone anywhere (and we can just add back later if needed).